dc/ref-dc-aias-aproc-proc.yaml

```diff
@@ -64,4 +64,4 @@ services:
     volumes:
       - ${APROC_INPUT_DIR}:/inputs:ro
       - ${APROC_DOWNLOAD_DIR}:/outbox
-      - ${PWD}/conf/aias/drivers.yaml:/app/conf/drivers.yaml:ro
+      - ${PWD}/conf/aias/drivers.yaml:/home/app/worker/conf/drivers.yaml:ro
```

Comment:
is that normal this addition, which is not the same as in FAM's docker file?

Reply:
yes, it's because it is a celery worker.